completions/ip

```diff
@@ -384,12 +384,30 @@ _comp_cmd_ip()
                 add | identify | list-id)
                     # TODO
                     ;;
-                delete | exec | pids | set)
+                delete | pids | set)
                     [[ $prev == "$subcmd" ]] && _comp_cmd_ip__netns "$1"
                     ;;
+                exec)
+                    local is_all=0
```

Comment:
The typical way to handle boolean flags like this in the codebase is to initialize to empty (when false is desired as the default), then set it to `set` on true, and check with `if [[ $is_all ]]`. grepping for `has_` yields some examples.

Ah, but reading a bit further, for this case, I see the value is actually being used as an integer, so renaming it to something more descriptive would be in order.

Reply:
Changed the name to all_offset, does that seem fine?

Reply:
Yes :+1: